index.js

```diff
@@ -41,3 +41,77 @@ app.get('/dogs', (req, res) => {
         })
     })
 })
+
+app.post('/dogs/:id/win', jsonParser, (req, res) => {
+    let id = req.param('id')
+    let regex = RegExp('[0-9a-fA-F]{24}')
+    if (regex.exec(id) == null) {
+        return res.status(400).json({
+            success: false,
+            message: 'Please ensure request has valid "winnerID" and try again.',
+            data: []
+        })
+    }
+    mongoClient.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true }, (err, client) => {
+        if (err) {
+            return res.status(500).json({
+                success: false,
+                message: 'Error connecting to database.',
+                data: []
+            })
+        }
+        let db = client.db(dbName)
```

Comment:
I think we get this `db`, then all we do is pass it into `declareChampion` and then do 
`let collection = db.collection(collectionName)`. We don't need the db variable otherwise so we should just create the collection and pass it into the function

Reply:
on it